rmarkdowns/Week_4.Rmd

```diff
@@ -257,10 +260,261 @@ For your challenges this week, create the necessary code chunks below. You can c
+  theme_linedraw() +
+  labs(
+    color="Common Name", #this changes the title of our legend
+    x = "Longitude",
+    y = "Latitude"
+  ) 
+
+plot
+```
+
+
+
+```{r}
+
+# Try again.... 
+# What I did differently - I started with the main code first geom_point(data = combined_df...... then print plot. Slowly add 1 layer at a time i.e.  geom sf(data = Oahu_streets, etc), print plot, then add more layer, print plot, etc. 
+
+# Result - Much better... :) 
+# Lesson Learned - "Never give up - Never surrender". 
+
+# Summary of Analysis - The majority of Mamaki and Noni medicinal plants are outside of protected areas in Oahu, Hawaii (assuming all the protected area have been added to Open Street Map)
+
+
+plot1 <- ggplot() +
+    geom_sf(data = Oahu_streets$osm_lines, inherit.aes = FALSE, color = "#1b9e77", size = .4, alpha = .8) +
+  geom_sf(data = Oahu_small_streets$osm_lines, inherit.aes = FALSE, color = "#d95f02", size = .2, alpha = .8) +
+    geom_sf(data = Oahu_coast$osm_lines, inherit.aes = FALSE, color = "#7570b3", size = .8, alpha = .5) +
+    geom_sf(data = Oahu_np$osm_polygons, inherit.aes = FALSE, color = "#e7298a", size = .2, alpha = .8) +
+    geom_sf(data = Oahu_protected$osm_polygons, inherit.aes = FALSE, color = "#66a61e", size = .2, alpha = .8) +
+  geom_sf_text(size = 1, data = Oahu_protected$osm_polygons, aes(label = name)) + #here we are adding some labels for our protected areas for context 
+  geom_point(data = combined_df, aes(x = longitude, y = latitude, color = common_name), size = 3) +
+  scale_shape_manual(values = c(21,22)) + 
+  scale_color_manual(values=c("#8dd3c7", "#bebada"))+
+  theme_grey() +
+  labs(title = "Mamaki (Piper Methysticum) and Noni (Morinda Citrifolia) *Medicinal Plants* in Oahu, Hawaii", 
+       subtitle = "Based on iNaturalist Data as of October 14, 2024") +
+  coord_sf(xlim = c(-158.28058, -157.64863), ylim = c(21.25482, 21.71201), expand = TRUE)
+
+plot1
+```
+
+```{r}
+# To add to plotly 
+ggplotly(plot1,
+         tooltip = c("text"))
+```
+
+```{r}
+
+# Objective = to examine the map of Waianae District to identify the presence of Mamaki and Noni medicinal plant.  Waianae District holds a special place in my heart due to its large houseless population and the insufficient number of medical centers and clinics available to support the community's health and wellness needs. 
+
+# Summary of Analysis - 1 Mamaki and 0 Noni plants in Waianae District.  This can be due to low number of observer personnel in Wainae District who input the data into iNaturalist website, etc.)
+
+# Circle Marker Color Legend - Purple (Mamaki) and Light Blue (Noni)
+
+# Future Work - From coding perspective, it would be nice to manually select the Circle Marker Color for Mamaki and Noni plants. However, given that I have already spent over 6+ hours on this week's assignment, it is time to finalize the current work and move on. 
```

Comment:
Good choice! Love the way you are thinking about this, but glad you decided to move forward. You can think about this for your final project, though!